### PR TITLE
[provider-ui] - Add Integration test for Provider UI

### PR DESCRIPTION
**Pull request by ramrodo**
**Description**

This change adds the first Integration Test for `provider-ui` and can be used as a start point for creating more. This test visits the root provider-ui page and validates if there is the main component.

It also adds `eslint-plugin-cypress` config to `eslint` for writing future tests. 

This PR is related to #1037

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Wow. That was fast.

**Comment by ramrodo** [comment]
> Wow. That was fast.

I started working on this last Monday.

File: provider-ui/.eslintrc.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ module.exports = {
     },
     "extends": [
         "eslint:recommended",
-        "plugin:react/recommended"
+        "plugin:react/recommended",
```
Comment (kushthedude) [inline provider-ui/.eslintrc.js:14]: Everything looks great! Can you please also update the eslint configuration of `/ui` folder?

**Review by kushthedude — COMMENTED** [review]
Great work @ramrodo !!

File: provider-ui/.eslintrc.js (hunk 1 above)
Comment (ramrodo) [reply provider-ui/.eslintrc.js:14]: Sure.

**Review by kushthedude — APPROVED** [review]

**Comment by kushthedude** [comment]
Congrats on your PR in meshery @ramrodo !!

**Comment by ramrodo** [comment]
> Congrats on your PR in meshery @ramrodo !!

Thanks @kushthedude!

**Comment by leecalcote** [comment]
Whoo-hoo! @ramrodo 🎈

**Pull request merged**